src/core/public/overlays/overlay_service.ts

```diff
@@ -75,19 +58,8 @@ export class OverlayService {
 export interface OverlayStart {
   /** {@link OverlayBannersStart} */
   banners: OverlayBannersStart;
-  openFlyout: (
-    flyoutChildren: React.ReactNode,
-    flyoutProps?: {
-      closeButtonAriaLabel?: string;
-      'data-test-subj'?: string;
-    }
-  ) => OverlayRef;
-  openModal: (
-    modalChildren: React.ReactNode,
-    modalProps?: {
-      className?: string;
-      closeButtonAriaLabel?: string;
-      'data-test-subj'?: string;
-    }
-  ) => OverlayRef;
+  /** {@link OverlayFlyoutStart#open} */
+  openFlyout: OverlayFlyoutStart['open'];
+  /** {@link OverlayModalStart#open} */
+  openModal: OverlayModalStart['open'];
```

Comment:
Are we going to make these APIs a bit more consistent with `banners` in a separate change? (eg. `overlays.flyout.open`)

Reply:
I did, but reverted it in https://github.com/elastic/kibana/pull/48431/commits/e26120b2cdf3c29e2c57492e9bd76323b03d807c following discussion with @rudolf starting at https://github.com/elastic/kibana/pull/48431#issuecomment-546347786 as there was no consensus. What should we do?